docs/source/api/express-middleware.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "API Reference: expressMiddleware"
+api_reference: true
+---
+
+This API reference documents Apollo Server 4's [Express](https://expressjs.com/) integration, the `expressMiddleware` function.
+
+## `expressMiddleware`
+
+The `expressMiddleware` function enables you to attach Apollo Server to an Express server. 
+
+The `expressMiddleware` function accepts two arguments, the first **required** argument is an instance of `ApolloServer` that has been `start`ed:
+
+<MultiCodeBlock>
+
+```ts
+import { ApolloServer } from '@apollo/server';
+import { expressMiddleware } from '@apollo/server/express4';
+import cors from 'cors';
+import { json } from 'body-parser';
+import express from 'express';
+
+const app = express();
+
+const server = new ApolloServer<MyContext>({
+  typeDefs,
+  resolvers,
+});
+// Note you must call `server.start()` on the `ApolloServer`
+// instance before passing the instance to `expressMiddleware`
+await server.start();
+
+// Specify the path where we'd like to mount our server
+//highlight-start
+app.use(
+  '/graphql',
+  cors<cors.CorsRequest>(),
+  json(),
+  expressMiddleware(server),
+);
+//highlight-end
+``` 
+
+</MultiCodeBlock>
+
+> ⚠️ To ensure your server gracefully shuts down, we recommend using the [`ApolloServerPluginDrainHttpServer`](/apollo-server/api/plugin/drain-http-server) plugin. [See below for an example](#example).
+
+The `expressMiddleware` function's second optional argument is an _object_ for configuring `ApolloServer`, which can contain the following option:
+
+<table class="field-table">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+
+<td>
+
+  ##### `context`
+
+  `Function`
+
+</td>
+
+<td>
+
+An optional asynchronous [`context` initialization function](../data/resolvers#the-context-argument). 
+  
+The `context` function should return an object, which gets passed to every resolver that executes for a particular operation. This enables resolvers to share helpful context values, such as a database connection.
```

Comment:
Is this first comma correct grammar?

Reply:
Nice catch! I also tweaked this sentence a bit:

> The `context` function should return an object shared across all resolvers that are executing for a particular operation.